packages/oui-datagrid/datagrid.less

```diff
@@ -162,9 +162,15 @@
 
   &__slot {
     flex: 1 0;
+    display: inline-block;
 
     oui-action-menu {
       margin-bottom: rem-calc(10);
     }
   }
+
+  &__refresh {
+    margin-bottom: 0.625rem;
+    float: right;
```

Comment:
Isn't the siblings components using a flex display ? I think we could avoid having to use `float`, since the transclude slot has `flex: 1`. If necessary, the refresh button could be included in the `oui-criteria` component to control its positioning :)